.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Tests
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    # Run automatically at 8AM PST Monday-Friday
+    - cron: '0 15 * * 1-5'
```

Comment:
GH actions might have a concurrent job limit and maybe it's a good idea to not kick off all the jobs across twilio and sendgrid at the same time

Reply:
GitHub Enterprise has [180 concurrent jobs](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration) I don't think we have to worry about this yet